libr/core/cmd_print.c

```diff
@@ -2804,6 +2807,31 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 						as->block[p].strings);
 			}
 			break;
+		case 'e':
+			blockptr = malloc (ate - at);
+			if (!blockptr) {
+				eprintf ("Error: failed to malloc memory");
+				return 0;
+			}
+			if (!r_io_read_at (core->io, at, blockptr, (ate - at))) {
+				eprintf ("Error: failed to read block");
+				free (blockptr);
+				return 0;
+			}
+			entropy = (ut8)(r_hash_entropy_fraction (blockptr, (ate - at)) * 255);
+			entropy = 9 * entropy / 200; // normalize entropy from 0 to 10
+			if (use_color) {
+				if (entropy >= 7) {
+					r_cons_print (Color_BGRED);
+				} else if (entropy >= 4) {
+					r_cons_print (Color_BGGREEN);
+				} else {
+					r_cons_print (Color_BGBLUE);
+				}
```

Comment:
put the r_cons_printf ("%s%d"Color_RESET, color);

Reply:
I couldn't make it work this way, don't really know what I'm doing wrong.

Reply:
show me the code

Reply:
It's already in the last commit: https://github.com/radare/radare2/commit/bd7b61df156643bf62c81a447d4f50ee320f9670

Reply:
'''
if (r_config_get_i (core->config, "scr.color")) {
			const char *color =
				(entropy > 6) ? Color_BGRED :
				(entropy > 3) ? Color_BGGREEN :
				Color_BGBLUE;
			r_cons_printf (Color_RESET"%s", color);
		}
		r_cons_printf ("%d", entropy);
'''